deps/build.jl

```diff
@@ -4,10 +4,10 @@ module DefaultDeps
     if isfile("deps.jl")
         include("deps.jl")
     end
-    if !isdefined(:ROOTENV)
+    if !@isdefined(ROOTENV)
```

Comment:
This will break Julia 0.6.   Maybe change it to `!isdefined(@__MODULE__, :ROOTENV)`